testing/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.15)
+
+set(SOURCE_FILES 
+    "configitemtest.cc"
+    "fixednotegraphscalertest.cc"
+    "notegraphscalerfactorytest.cc"
+    "utiltest.cc"
+)
+set(HEADER_FILES
+)
+set(GAME_SOURCES 
```

Comment:
here you should probaly not compile again thos sources but create a static lib with them pulling the correct dependencies (boost::local and fmt)

Reply:
I would like to do that in a following PR just to get a start for the testing stuff.